crates/pixi_build_frontend/src/protocols/builders/rattler_build.rs

```diff
@@ -134,15 +142,27 @@ impl ProtocolBuilder {
         tool: &ToolCache,
         build_id: usize,
     ) -> Result<JsonRPCBuildProtocol, FinishError> {
-        let tool_spec = self
-            .backend_spec
-            .ok_or(FinishError::NoBuildSection(self.manifest_path.clone()))?;
+        let tool_spec = self.backend_spec.unwrap_or_else(|| {
+            ToolSpec::Isolated(IsolatedToolSpec::from_specs(["pixi-build-rattler-build"
```

Comment:
Might be nice to use a constant for `pixi-build-rattler-build` like `RECIPE_BACKEND` or something :)

Reply:
Also now that I'm reading it `pixi-build-recipe` might make more sense as a backend name as well, wdyt? cc @nichmor